### ISSUE TITLE
Add ACV Support

### ISSUE THREAD

**Issue by efremidze** [BradLarson/GPUImage2#43]
I don't see any support for ACVs.
`- (id)initWithACV:(NSString*)curveFilename;`

**Comment by BradLarson** [comment]
Correct. That particular operation hasn't yet been ported over from the Objective-C code.

**Comment by efremidze** [comment]
Ill make a PR

**Comment by harmjanr** [comment]
@efremidze Did you ever finish that PR? I need GPUImage in swift with ACV in one of my projects as well, and really don't want to use the ObjC lib with a bridging header :)

**Comment by ShivaHuang** [comment]
I think using the objC version is a better choice. The Swift version lacks lots of functions and has stopped developing nor accepting PRs.

The objC version is old, needs bridging, but it works.

**Comment by harmjanr** [comment]
@ShivaHuang We'll try that. Thanks!
